docs/production/authentication-methods.md

```diff
@@ -669,6 +669,9 @@ Logout endpoint with a `LogoutRequest`. If a successful
 `LogoutResponse` is received back, their current Zulip session will be
 terminated.
 
+Note that this doesn't work in the case of desktop and mobile application
+and is reserved to the browser.
+
```

Comment:
I'd squash this back into the commit that added the functionality -- particularly since later commits are going to fix it, it doesn't seem worth highlighting as its own commit.